presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -884,6 +884,17 @@ This can also be specified on a per-query basis using the ``confidence_based_bro
 Enable treating ``LOW`` confidence, zero estimations as ``UNKNOWN`` during joins. This can also be specified
 on a per-query basis using the ``treat-low-confidence-zero-estimation-as-unknown`` session property.
 
+``optimizer.scalar-function-stats-propagation-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+Enable scalar functions stats propagation using annotations. Annotations define the behavior of the scalar
+function's stats characteristics. When set to ``true``, this property enables the stats propagation through annotations.
+This can also be specified on a per-query basis using the ``scalar_function_stats_propagation_enabled`` session property.
```

Comment:
I think this is better than before. However, users still shouldn't know/care that a function is annotated or even that annotations exist. It is an implementation detail. How about:
```suggestion
When set to ``true``, this property enables statistics to be propagated from function arguments to higher level expressions. When set to ``false`` any function calls result in unknown statistics which can negatively impact query planning.
This can also be specified on a per-query basis using the ``scalar_function_stats_propagation_enabled`` session property.
```